lib/processes/proxy/proxy.js

```diff
@@ -107,11 +111,10 @@ function proxyRequest(request, response, toPort) {
   console.log('--> %s %s', request.method, request.url);
 
   var remoteRequestOptions = buildRemoteRequestOptions(request, toPort);
-  console.log('    %j', remoteRequestOptions);
-
   if (isNewPage(request.url)) {
     modifyRequest(remoteRequestOptions, newPageOptions);
   }
+  console.log('    %j', remoteRequestOptions);
```

Comment:
Moved so the additional options are correctly logged. Makes finding issues like this much easier.